src/net.h

```diff
@@ -838,37 +838,89 @@ class CNode
         assert(false);
     }
 
+    /* These are the default connections that we use to connect with the
```

Comment:
These are great comments. Thank you! Should they be merged with the comments above on the enum members?

Reply:
yeah I felt very indecisive when trying to figure out the best spots for the comments. here what I've gone for is having the enum comments be a more concise description of the nature of the connection, and the function provide more context around design goals, capacity, etc. 

I don't think the split is super clear though, so yeah, maybe it would be more helpful to consolidate it all into the enum member comments. is that what you're advocating for? or just trying to weigh like me? ⚖️

Reply:
yes, consolidate them to one place. I think the enum is the right place for them.

Reply:
done!